2-build-msys-cmake.sh

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env bash
+
+export PKG_CONFIG_PATH="${PKG_CONFIG_PATH}:C\:/msys64/opt/mlt-6.16.0/lib/pkgconfig"
```

Comment:
Since the real path can be for example `d:\msys`, it is better to use `/opt/mlt-6.16.0/lib/pkgconfig`

Reply:
(these were some test changes I was trying, to build something on my system and over here simultaneously)

But then, you asked me to remove the prefix part from CMake.